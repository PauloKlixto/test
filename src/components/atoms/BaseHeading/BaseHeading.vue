<template>
  <section class="BaseHeading">
    <h1>{{ label }}</h1>
  </section>
</template>

<script>
export default {
  name: 'BaseHeading',
  props: {
    label: {
      type: String,
      default: ''
    }
  }
}
</script>
