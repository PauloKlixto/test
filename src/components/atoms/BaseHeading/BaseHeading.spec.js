/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils'
import BaseHeading from './BaseHeading'

describe('BaseHeading', () => {
  const WRAPPER = shallowMount(BaseHeading)

  test('Componente existe', () => {
    expect(WRAPPER.exists()).toBe(true)
  })

  test('Componente nÃ£o alterado', () => {
    expect(WRAPPER.html()).toMatchSnapshot()
  })
})
