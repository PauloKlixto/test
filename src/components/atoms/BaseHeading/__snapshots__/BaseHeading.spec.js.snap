// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseHeading Componente n√£o alterado 1`] = `
"<section class=\\"BaseHeading\\">
  <!---->
  <h1></h1>
</section>"
`;
